packages/console/src/components/Breakable/index.module.scss

```diff
@@ -0,0 +1,5 @@
+.breakable {
+  word-break: break-word;
+  white-space: pre-wrap;
+  overflow-wrap: break-word;
+}
```

Comment:
Maybe an scss mixin would be more suitable for this?

Reply:
thought about this, but mixin is not that straight-forward comparing to a React component. you can see the usage in this PR, some of them are pure text or children, which needs to define a new style file and add mixin to achieve so:

```tsx
<div className={styles.foo}>{title ?? value}</div>
```

```scss
.foo {
  @use _.breakable;
}
```

which equals to the component implementation in this PR

Reply:
discussed offline